apps/web/src/pages/Applications/ApplicantDashboardPage/components/DashboardHeading.tsx

```diff
@@ -100,6 +106,100 @@ const DashboardHeading = ({ user }: DashboardHeadingProps) => {
         description: "Subtitle for applicant dashboard hero",
       })}
     >
+      {searchParams.get(FromIapDraftQueryKey) === "true" && (
+        <Alert.Root
+          type="info"
+          dismissible
+          onDismiss={() => {
+            searchParams.delete(FromIapDraftQueryKey);
+            setSearchParams(searchParams);
```

Comment:
Should we add a `{replace: true}` as the second arg here? Right now if you hit the browser back button after dismissing the alert, it just shows up again. This may be desirable though 🤔

Reply:
I didn't consider that this would add an entry to history.  I added the `replace` and I like how that works.  Honestly I'm not sure we need to mutate the search params at all.  I could be convinced to remove the onDismiss entirely.
Ref: [replace search state](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6750/commits/88bcd764e5c20cdede959910f90c6f09ca3579f0)